runbotics/runbotics-desktop/src/action/folder/folder.test.ts

```diff
@@ -71,14 +71,21 @@ describe('FolderActionHandler', () => {
         }
     };
 
+    const removeTestFolder = () => {
+        const fullPath = `${tempFolderPath}${testFolderName}`;
+        if (fs.existsSync(fullPath)) {
+            fs.rmSync(fullPath);
+        }
```

Comment:
else throw error "could not remove test folder"?

Reply:
those are tests, if it wasn't there than not a problem. Its just cleanup function